paddle/go/optimizer/Tensor.h

```diff
@@ -0,0 +1,25 @@
+#ifndef PADDLE_FAKE_TENSOR_H_
+#define PADDLE_FAKE_TENSOR_H_
+/**
+ * @brief fake tensor for testing 
+ */
+
+#include "paddle/math/BaseMatrix.h"
+#include <string.h>
+
+namespace paddle {
+template <class T>
```

Comment:
since this is the tensor only used by optimizer for now, maybe add namespace pserver here. e.g., `paddle::optimizer::Tensor`